doc/source/programs/ogr_layer_algebra.rst

```diff
@@ -0,0 +1,169 @@
+input source , a method source and generates the output of the operation in the specified output file
+
+.. program:: ogr_layer_algebra
+
+.. option:: <mode>
+
+    Where <mode> is one of the seven available modes:
+
+    * ``Union``
+
+        A union is a set of features, which represent areas that are in either of the operand layers. 
+
+    * ``Intersection``
+
+        An intersection is a set of features, which represent the common areas of two layers. 
+
+    * ``SymDifference``
+
+        A symmetric difference is a set of features, which represent areas that are in operand layers but which do not intersect. 
+
+    * ``Identity``
+
+        The identity method identifies features in the input layer with features in the method layer possibly splitting features into several features. 
+        By default the result layer has attributes from both operand layers. 
+
+    * ``Update``
+
+        The update method creates a layer, which add features into the input layer from the method layer possibly cutting features in the input layer. 
+        By default the result layer has attributes only from the input layer. 
+
+    * ``Clip``
+
+        The clip method creates a layer, which has features from the input layer clipped to the areas of the features in the method layer. 
+        By default the result layer has attributes of the input layer. 
+
+    * ``Erase``
+
+        The erase method creates a layer, which has features from the input layer whose areas are erased by the features in the method layer. 
+        By default the result layer has attributes of the input layer. 
+        
+.. option:: -input_ds <path>
+
+    Input data set path for the operation to be performed.
+     For operations involving 2 datasets, this is one of the dataset.
+
+.. option:: -input_lyr <name>
+
+    Layer name of the ``input_ds`` for which the operations have to be performed ( Optional )
+
+.. option:: -method_ds <path>
+
+    Method data set path for the operation to be performed. 
+    This is usually the conditional data set supplied to the operation ( ex: clip , erase , update )
+    This is the Second data set in the operation ( ex : Union, Intersection , SymDifference )
+
+.. option:: -method_lyr <name>
+
+    Layer name of the ``method_ds`` for which the operations have to be performed ( Optional )
+
+.. option:: -output_ds <path>
+
+    Output data set path for writing the result of the operations performed by ogr_layer_algebra
+
+.. option:: -output_lyr_name <name>
+
+    Layer name of the ``output_lyr_name`` where the output vector has to be written. ( Optional )
+
+.. option:: -overwrite
+
+    Indicates wether the ``output_ds`` have to be overwritten with the generated result of ogr_layer_algebra
+
+.. option:: -opt <NAME=VALUE>
+
+    Attributes for which the operation has to run on ``input_ds`` and ``method_ds``
+
+.. option:: -f <format_name>
+
+    Select the output format. Starting with GDAL 2.3, if not specified,
```

Comment:
```suggestion
    Select the output format. If not specified,
```

Reply:
Added this in https://github.com/kssvrk/gdal/commit/0a8393cee671b57f498c051b3ba83a1c5401f6c4